packages/design-system-docs/src/pages/utilities/layout-grid/_grid.docs.scss

```
@@ -94,17 +69,39 @@ Align columns horizontally or vertically using [flexbox utility classes]({{root}
 
 Markup: grid-alignment.example.html
 
-Style guide: styles.grid.alignment
+Style guide: utilities.grid.alignment
 */
 
 /*
-Forms
+Form rows
 
 Use a `.ds-l-form-row` class to tighten the column spacing when laying out form fields in a grid.
 
 Markup: grid-forms.example.html
 
-Style guide: styles.grid.form-row
+Style guide: utilities.grid.form-row
+*/
+
+/*
+Responsive grids
+
+@responsive
+
+Use a [breakpoint prefix]({{root}}/guidelines/responsive) to change the column widths based on the viewport width.
+
+Responsive format: .ds-l-[breakpoint]-col--*
+
+In the example below, the cells span:
+
+- 1/6 of the row on `xl` viewports, using `.ds-l-xl-col--2`
```

This fraction based terminology is a bit confusing for me. 
What about something like this for these explanations? 

```suggestion
- 6 cells per row on `xl` viewports, using `.ds-l-xl-col--2`
```

---

That does read better